### PR TITLE
Rspec tests for package branching

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

**Comment by bgeuken** [comment]
@hennevogel please review

File: src/api/spec/factories/packages.rb [hunk 1]
```diff
@@ -3,7 +3,12 @@
     sequence(:name) { |n| "#{Faker::Internet.domain_word}#{n}" }
     factory :package_with_file do
       after(:create) do |package|
-        Suse::Backend.put("/source/#{package.project.name}/#{package.name}/somefile.txt", Faker::Lorem.paragraph)
+        CONFIG['global_write_through'] = true
```
Comment (hennevogel) [inline src/api/spec/factories/packages.rb]: Again?

Comment (bgeuken) [inline src/api/spec/factories/packages.rb]: Argh. Forgot to remove that after creating the VCR cassettes. Good catch!

Comment (bgeuken) [inline src/api/spec/factories/packages.rb]: fixed

**Comment by bgeuken** [comment]
Just noticed that I forgot to cleanup spec/features/webui/packages_spec.rb from the global write through flags. I did that now.

**Comment by bgeuken** [comment]
@hennevogel ? go or no go?

**Comment by hennevogel** [comment]
giphy

LGTM

**Pull request merged**